.evergreen/config.yml

```diff
@@ -475,6 +475,10 @@ tasks:
     commands:
       - func: "export FLE credentials"
       - func: "run tests"
+  - name: "test-fle-via-docker"
+    commands:
+      - func: "export FLE credentials"
+      - func: "run tests via docker"
```

Comment:
This should have been part of https://github.com/mongodb/mongo-ruby-driver/pull/2395, my bad.